release-notes/opensearch-security-dashboards-plugin.release-notes-1.2.0.0.md

```diff
@@ -5,7 +5,10 @@
 *  Add observability cluster permissions ([#851](https://github.com/opensearch-project/security-dashboards-plugin/pull/851))
 
 ### Maintenance
+
```

Comment:
If there is no empty row after `### Enhancements`, then this empty row is not needed.

Reply:
fixed